src/v/pandaproxy/schema_registry/test/BUILD

```diff
@@ -0,0 +1,229 @@
+load("//bazel:test.bzl", "redpanda_cc_btest", "redpanda_test_cc_library")
+
+redpanda_test_cc_library(
+    name = "compatibility_avro",
+    hdrs = [
+        "compatibility_avro.h",
+    ],
+    include_prefix = "pandaproxy/schema_registry/test",
+    deps = [
+        "//src/v/pandaproxy",
+    ],
+)
+
+redpanda_test_cc_library(
+    name = "compatibility_protobuf",
+    hdrs = [
+        "compatibility_protobuf.h",
+    ],
+    include_prefix = "pandaproxy/schema_registry/test",
+    deps = [
+        "//src/v/pandaproxy",
+    ],
+)
+
+redpanda_test_cc_library(
+    name = "compatibility_common",
+    hdrs = [
+        "compatibility_common.h",
+    ],
+    include_prefix = "pandaproxy/schema_registry/test",
+    deps = [
+        "//src/v/pandaproxy",
+    ],
+)
+
+redpanda_cc_btest(
+    name = "sanitize_avro",
+    timeout = "short",
+    srcs = [
+        "sanitize_avro.cc",
+    ],
+    deps = [
+        "//src/v/pandaproxy",
+        "//src/v/pandaproxy/schema_registry/test:compatibility_avro",
```

Comment:
nitpick (here and elsewhere): I think this could be

```suggestion
        ":compatibility_avro",
```

Reply:
ahh yeh i think so. i used a script to auto-generate these

Reply:
no but i posted it in the bazel channel. it's kinda hard to use, but it's been useful for this temporary backfill project